src/main/java/seedu/address/model/module/ModuleHasModuleCodePredicate.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.module;
```

Comment:
perhaps can package this into a `seedu.address.model.module.predicates` package? this way we can keep the 'root' path of model.module cleaner